client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -59,7 +63,7 @@
 
 final class Request {
 
-    private static final String DELIMITER = "/";
+    private static final XContentType CONTENT_TYPE = XContentType.JSON;
```

Comment:
it is supposed to be used in all requests that have a body which we generate calling toXContent. It is not just about search (although it is at the moment)